### ISSUE TITLE
Configuration error in hugo？

### ISSUE THREAD

**Issue by zburu** [CloudCannon/pagefind#96]
I had some issues with hugo, running `npx pagefind --source "public"`, then the terminal prompts `x pagefind_extended`

Below is my configuration, do I have a problem somewhere?

````html
<link rel="stylesheet" href="{{ .Site.Params.staticPrefix }}{{ "_pagefind/pagefind-ui.css" | relURL }}">
<script src="{{ .Site.Params.staticPrefix }}{{ "_pagefind/pagefind-ui.js" | relURL }}"></script>
<div id="search"></div>
<script>
 window.addEventListener('DOMContentLoaded', (event) => {
   new PagefindUI({ element: "#search" });
 });
</script>
````

**Comment by bglw** [comment]
Hello! 👋 

I don't recognize that terminal prompt — can you add a screenshot of what shows exactly when you run that command? 

Also, what operating system / architecture are you using? 

Lastly, running `npm install pagefind` in a test directory might surface a better error than `npx pagefind` does, if some error is being hit.

**Comment by zburu** [comment]
This is a great project.

Sorry to disturb you due to the time difference, Here are some descriptions! I use windows computer!

**1.when executing the command**
![image](https://user-images.githubusercontent.com/65840178/196311925-a5744ba3-a58a-4bf4-beb3-c00ce6692873.png)

**2.after executing the command**
![image](https://user-images.githubusercontent.com/65840178/196311932-f73b8866-f320-47b4-afb5-4e6ba3b42957.png)

This is the description from the pagefind documentation.
>Now build your site to an output directory — this guide assumes that you’re running hugo and that your site is output to the public/ directory. Pagefind works with any set of static HTML files, so adjust these configurations as needed.

I imported the css and js as requested, called it, and ran `npx pagefind --source "public"` . But the result is `x pagefind_extended`, I don't quite understand what it means to find a result?

But the result is that building the index fails, I don't know what to do, I hope you can give me some advice

**Comment by bglw** [comment]
No worries — happy to help!

Hmm, I think this might the same issue as #66 which is a Windows issue I haven't had a chance to test the fix for yet.

I'll endeavor to look at this very soon — in the meantime if you download the Windows release directly from https://github.com/CloudCannon/pagefind/releases/tag/v0.8.1 and rename it to `pagefind_extended.exe` you should be able to run it from your terminal.

Keep you posted — sorry for the delay in fixing this!

**Comment by bglw** [comment]
👋 Hello! This is fixed in the [Pagefind v0.9.1](https://github.com/CloudCannon/pagefind/releases/tag/v0.9.1) release 🎉
